lib/components/gene/__tests__/about-test.js

```diff
@@ -0,0 +1,37 @@
+import 'react-native'
+import React from 'react'
+import About from '../about'
+
+// Note: test renderer must be required after react-native.
+import renderer from 'react-test-renderer'
+
+jest.mock('../../opaque_image_view.js', () => 'ImageView')
+
+it('renders correctly', () => {
+  const gene = {
+    description: 'This is some example text',
+    trending_artists : []
+  }
+  const about = renderer.create(<About gene={gene}/>).toJSON()
+  expect(about).toMatchSnapshot()
+})
```

Comment:
We can effectively ignore Relay for this, we are just using the props directly ( not triggering a Relay API grab ( due to [this file](https://github.com/artsy/emission/blob/master/lib/__mocks__/react-relay.js) ) )